nautobot/ui/jest.config.js

```diff
@@ -0,0 +1,43 @@
+const nautobotAlias = require("./aliases")
```

Comment:
To implement nautobotAlias, I extract the jest configuration from package.json and paste it here.